### ISSUE TITLE
Question about missing encoder_attention_mask in FluxTransformer2DModel training

### ISSUE THREAD

**Issue by IQIUM** [declare-lab/TangoFlux#13]
Hello, I am a beginner. I compared the implementations of tango and tangoflux repositories. In the training code of tangoflux, I noticed a difference when using FluxTransformer2DModel for prediction - it does not pass parameters like encoder_attention_mask=boolean_encoder_mask. I understand that FluxTransformer2DModel doesn't have this parameter, but I want to know if this would cause any issues during training? 

tango implementation:
https://github.com/declare-lab/tango/blob/310e68504c3f91200313ccc3b94bf19e0941e339/models.py#L182-L185

tangoflux implementation:
https://github.com/declare-lab/TangoFlux/blob/8ec5b6d1e2c54f134810c5ed7cf4fa966dda34ba/tangoflux/model.py#L487-L497

**Comment by hungchiayu1** [comment]
Hi! Thanks for noticing this, during the implementation we realize FluxTransformer2DModel does not have attention mask so we did not pass in attention mask during training. Base on our knowledge(might be wrong), the original stable diffusion training does not use attention mask during training too, so we simply went ahead without attention mask. We did not notice any issue without attention mask during training.
